### ISSUE TITLE
Not working when using TypeScript in Svelte

### ISSUE THREAD

**Issue by oekazuma** [markuplint/markuplint#444]
### Describe the bug
markuplint does not work when using TypeScript in Svelte.

```shell
❯ npm run lint:markup
$ markuplint **/*.svelte
<markuplint> passed /home/projects/vitejs-vite-jgbxyk/src/App.svelte
<markuplint> error: Unexpected token
1: <script lang="ts">
2:   let count: number = 0
              ^
3:   const increment = () => {
4:     count += 1 (parse-error) /home/projects/vitejs-vite-jgbxyk/src/lib/Counter.svelte:2:11
```

### Reproduction
https://stackblitz.com/edit/vitejs-vite-jgbxyk

### Additional Information
Perhaps the problem is that you are passing the template as is to the svelte parser.
It might be better to preprocess or not to parse script and style.

Maybe the implementation of `prettier-plugin-svelte` could be helpful?
https://github.com/sveltejs/prettier-plugin-svelte/blob/009cda5c12ee09e188b5d3e67a520089c41b34c1/src/lib/snipTagContent.ts#L3

**Comment by YusukeHirao** [comment]
@oekazuma Thank you for reporting.

I will fix it. Please, wait a moment.
